android/src/main/java/com/swmansion/rnscreens/ScreenStackHeaderConfigViewManager.java

```diff
@@ -89,9 +89,9 @@ public void setTitleColor(ScreenStackHeaderConfig config, int titleColor) {
     config.setTitleColor(titleColor);
   }
 
-  @ReactProp(name = "backgroundColor", customType = "Color")
-  public void setBackgroundColor(ScreenStackHeaderConfig config, int titleColor) {
-    config.setBackgroundColor(titleColor);
+  @ReactProp(name = "backgroundColor", customType = "Color", defaultInt = -1)
```

Comment:
@andreibarabas I am noticing that when I try to set my header background color to "#fff" or "#ffffff" (i.e. white), I end up with a non-white header. If you take a look at the attached photo, you can see the header appears to be gray. Also note that the status bar at the top is white.

Any chance these bugs came in with the changes to this file?

<img width="387" alt="Screen Shot 2020-10-04 at 12 12 00 PM" src="https://user-images.githubusercontent.com/30055476/95024906-d0730200-063a-11eb-8ba5-50c45cd5c682.png">

Reply:
To clarify, this is when `headerTranslucent` is set to false.

Reply:
After some further testing, @andreibarabas it looks like it was the change in ScreenStackHeaderConfig.java

Reply:
Looks like the "#fff", "#ffffff" and `white` colors all have value of "-1" on the native side, so the newly added if makes it impossible to use these values. Can you check if https://github.com/software-mansion/react-native-screens/pull/656 fixes the issue?

Reply:
@WoLewicki that fixed the issue! Thanks

Reply:
Guys, glad you solved it. Thanks! Just now had the time to groom the notifications.